src/resources/postcss/tribe-events-single-blocks-reskin.pcss

```diff
@@ -0,0 +1,51 @@
+/**
+ * Block editor event single v2 reskin
+ *
+ * @since TBD
+ */
+
+/* = Website block view styles
+============================================= */
+.tribe-block__event-website {
+	padding: 20px 0;
+
+	a {
+		background-color: var(--tec-color-link-accent);
+		border: none;
+		border-radius: 4px;
+		font-size: 1.125rem;
+		font-weight: var(--tec-font-weight-bold);
+		min-height: 40px;
+		padding: 0 20px;
+	}
+}
+
+#tribe-events-content .tribe-block__event-website a {
```

Comment:
you can include this in the above, doing something like:

```
.tribe-block__events-website {

    a {

        #tribe-events-content & {
            // styles
        }
    }
}
```

Reply:
Updated.